### PR TITLE
fix: Implemented organizer access for attendee tickets

### LINKED ISSUE #5968: Organizer should be able to download tickets of attendees
Organizers should have access to download tickets of attendees of their own events.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #5968

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Implemented organizer access for attendee tickets

#### Changes proposed in this pull request:

- Pulled organizer ids and checked if the current user was the organizer or the event attendee

### CODE REVIEW

**Comment by mrsaicharan1** [comment]
@uds5501 @shreyanshdwivedi @iamareebjamal Please check.

File: app/api/helpers/events.py [hunk 1]
```diff
@@ -28,3 +28,11 @@ def create_custom_forms_for_attendees(event):
     save_to_db(first_name_form, 'First name form saved')
     save_to_db(last_name_form, 'Last name form saved')
     save_to_db(email_form, 'Email form saved')
+
+
+def organizer_status(user, order):
```
Comment (iamareebjamal) [inline app/api/helpers/events.py]: Move this method to event model and rename to `is_organizer` and it'll take user ID as parameter. 

If there already is a method to check permissions, use it

Comment (mrsaicharan1) [reply app/api/helpers/events.py]: @iamareebjamal Used aa predefined `is_organizer`

File: app/api/auth.py [hunk 2]
```diff
@@ -283,22 +283,24 @@ def change_password():
         "password-changed": True
     })
 
-
+# helper methods
```
Comment (iamareebjamal) [inline app/api/auth.py]: Remove

Comment (mrsaicharan1) [reply app/api/auth.py]: @iamareebjamal Updated.

**Pull request merged**